pkg/tests/apis/provisioning/testdata/openapi.json

```diff
@@ -4396,309 +4436,7 @@
             "description": "Whether and how garbage collection will be performed. Either this field or OrphanDependents may be set, but not both. The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy. Acceptable values are: 'Orphan' - orphan the dependents; 'Background' - allow the garbage collector to delete the dependents in the background; 'Foreground' - a cascading policy that deletes all dependents in the foreground.",
             "type": "string"
           }
-        },
-        "x-kubernetes-group-version-kind": [
```

Comment:
See: https://github.com/grafana/grafana/pull/98578 -- this list grows as we have unrelated apis in the server